src/building/suggested.md

```diff
@@ -26,13 +26,14 @@ of these tools when hacking on `rustc`. For example, `x.py setup` will prompt
 you to create a `.vscode/settings.json` file which will configure Visual Studio code.
 This will ask `rust-analyzer` to use `./x.py check` to check the sources, and the
 stage 0 rustfmt to format them.
+The recommended `rust-analyzer` settings live at [`src/etc/vscode_settings.json`](https://github.com/rust-lang/rust/blob/master/src/etc/vscode_settings.json).
 
 If you have enough free disk space and you would like to be able to run `x.py` commands while
 rust-analyzer runs in the background, you can also add `--build-dir build-rust-analyzer` to the
 `overrideCommand` to avoid x.py locking.
 
 If you're running `coc.nvim`, you can use `:CocLocalConfig` to create a
-`.vim/coc-settings.json` and copy the settings from [this file](https://github.com/rust-lang/rust/blob/master/src/etc/vscode_settings.json).
+`.vim/coc-settings.json` and copy the settings from the above file.
```

Comment:
I would duplicate the link. You can use reference links to avoid writing it out twice.
```suggestion
`.vim/coc-settings.json` and copy the settings from [`src/etc/vscode_settings.json`].

[`src/etc/vscode_settings.json`]: https://github.com/rust-lang/rust/blob/master/src/etc/vscode_settings.json
```